package-lock.json

```diff
@@ -9,9 +9,9 @@
       "version": "1.3.8",
```

Comment:
Please update here too

Reply:
We have to fix this. Package-lock.json should be part of .gitignore. I've added it now.